### DISCUSSION TITLE
Controllo CAPTCHA su questionario

### DISCUSSION THREAD

**Discussion by etrix80** [globaleaks/globaleaks-whistleblowing-software#3750] [Q&A]
Salve a tutti, volevo sapere se è possibile attivare un controllo captcha o similare prima di poter inviare il questionario 
in modo da evitare segnalazioni spam da bot.

Grazie in anticipo

**Comment by evilaliv3** [comment]
@etrix80: A meno di non integrate tecnologie di terze parti quali quelle di recaptcha o google, I captcha di tipo grafico o numerico sono di fatto facilmente aggirabili.

Per questo motivo in GlobaLeaks implementiamo solo un captcha automatizzato volto a limitare gli attaccanti; a questo si aggiungono le protezioni date dall'implementazione di un buon questionario di segnalazione.

**Comment by etrix80** [comment]
@evilaliv3 postresti spiegarmi meglio come funziona questo captcha automatizzato e se devo abilitarlo oppure se è attivo di default?

**Comment by evilaliv3** [comment]
E' attivo di default.

Per poter attivare una segnalazione il browser deve eseguire prima una computazione matematica. (una serie di sha256  necessari a risolvere un problema probabilistico che gli fa impiegare del tempo computazionale)

La metodologia si chiama "proof of work"
